src/pages/billing/List/List.ts

```diff
@@ -118,11 +121,11 @@ export const List = (props: ListProps) => {
   })
 
   useEffect(() => {
-    const anyProjectsCreating = _.some(({ status }) => isCreatingStatus(status), billingProjects)
+    const anyProjectsSyncing = _.some(({ status }) => isSyncingStatus(status), billingProjects)
 
-    if (anyProjectsCreating && !interval.current) {
-      interval.current = window.setInterval(loadProjects, 10000)
-    } else if (!anyProjectsCreating && interval.current) {
+    if (anyProjectsSyncing && !interval.current) {
+      interval.current = window.setInterval(loadProjects, 20000)
```

Comment:
Increased the reload interval because we expect these operations to take about 15 minutes. If you are trying to do something like create a billing project at the exact time that the page reloads, it's annoying.

I could be persuaded to further increase the interval.

Reply:
I'd vote for 30 seconds but I'm fine with 20 too

Reply:
Increased to 30 seconds.